Sources/TuistServer/Session/ServerSessionController.swift

```diff
@@ -126,6 +146,21 @@ public final class ServerSessionController: ServerSessionControlling {
         }
     }
 
+    public func getAuthenticatedHandle(serverURL: URL) async throws -> String {
```

Comment:
```suggestion
    public func authenticatedHandle(serverURL: URL) async throws -> String {
```

The `getXXX` convention is rarely used in Swift, so I'd rename the method here.